src/polyhedralGravity/model/KDTree/KdDefinitions.h

```diff
@@ -39,6 +70,42 @@ namespace polyhedralGravity {
          * Specifies which coordinate dimension is fixed for every point on the plane.
          */
         Direction orientation;
+
+        /**
+        * Returns the normal vector for this plane.
+        * @param returnFlipped Whether to return the normal pointing in the opposite direction.
+        * @return The normal vector.
+        */
+        [[nodiscard]] Array3 normal(const bool returnFlipped = false) const {
+            using namespace util;
+            return polyhedralGravity::normal(orientation) * (returnFlipped ? -1 : 1);
+        }
+
+        /**
+         * Returns the origin point of a plane, meaning a point that lies on the plane.
+         * @return The origin point.
+         */
+        [[nodiscard]] Array3 originPoint() const {
+            Array3 point{};
+            point.fill(0);
```

Comment:
```suggestion
            Array3 point{0.0, 0.0, 0.0};
```

For a three element array, this is more concise.